packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -67,23 +67,25 @@ class FuchsiaSdk {
         globals.printError('Have you set FUCHSIA_SSH_CONFIG or FUCHSIA_BUILD_DIR?');
         return null;
       }
-      const String remoteCommand = 'log_listener --clock Local';
-      final List<String> cmd = <String>[
-        'ssh',
-        '-F',
-        fuchsiaArtifacts.sshConfig.absolute.path,
-        id,
-        remoteCommand,
-      ];
-      globals.processManager.start(cmd).then((Process newProcess) {
-        if (controller.isClosed) {
-          return;
-        }
-        process = newProcess;
-        process.exitCode.whenComplete(controller.close);
-        controller.addStream(process.stdout
-            .transform(utf8.decoder)
-            .transform(const LineSplitter()));
+      fuchsiaDevFinder.resolve(name).then((String resolvedIp) {
```

Comment:
We should probably just resolve the ip once-and-for-all when creating the `FuchsiaDevice` in `FuchsiaDevices.pollingGetDevices()`. That context is also already async, so it will avoid the unawaited `Future` here.

Reply:
👍